CFGNano_Sftw_HW1_Karishma_Thaladi.py

```diff
@@ -0,0 +1,132 @@
+"""
+
+TASK 1
+
+Write a class to represent a Cash Register.
+You class should keep the state of price total and purchased items
+
+Below is a starter code:
+------------------------
+1. you can add new variables and function if you want to
+2. you will NEED to add accepted method parameters where required.
+For example, method add_item probably accepts some kind of an item?..
+3. You will need to write some examples of how your code works.
+
+"""
+
+class CashRegister:
+    total_items = []
+    total_price = 0
+
+    def __init__(self, item, price):
+
+        # self.total_items = None #{'item': 'price'}
+        # self.total_price = 0
+        self.discount = 0.10
+        self.item = item
+        self.price = price
+
+    def add_item(self, item,price):
+        # pass
+        self.total_items.append(self.item)
+        self.total_price += self.price
+
+    def remove_item(self):
+        # pass
+        self.item.remove(self.item)
+        self.price.remove(self.price)
+
+
+    def apply_discount(self):
+        # pass
+        total_price_discount = self.total_price * self.discount
+        return total_price_discount
+
+    def get_total(self):
+        # pass
+        self.total_price += self.price
+        return total_price
+
+    def show_items(self):
+        # pass
+        self.total_items.append(self.total_items + self.item)
+        return total_items
+
+    def reset_register(self):
+        # pass
+        self.item = item.clear()
+        self.price = price.clear()
+        self.total_price = total_price.clear()
+        self.total_items = total_items.clear()
+        self.discount = discount.clear()
+
+    def show_details(self):
+        print("Item: ", self.item)
+        print("Price: ", self.price)
+        print("Discount: ", self.discount)
+        print("Total Price: ", self.total_price)
+        print("Total Items: ", self.total_items)
+
+
+# EXAMPLE code run:
+
+# ADD
+item1 = CashRegister('jeans',12.99)
+print(item1)
+print(item1.show_details())
+item2 = CashRegister('top',5.99)
+print(item2)
+print(item2.show_details())
+
```

Comment:
Good solution, but some of your methods don't work, like apply_discount and I can't seem to add other shopping items unless I create another instance. Have a look at the OOP answers we posted in slack, these will be helpful :)
-30